client/signup/steps/design-picker/index.jsx

```diff
@@ -215,10 +218,16 @@ export default function DesignPickerStep( props ) {
 	}
 
 	function renderDesignPicker() {
+		const intent = dependencies.intent;
+		let showDesigns = useFeaturedPicksButtons ? designs : [ ...featuredPicksDesigns, ...designs ];
+		//Show only e-commerce themes for the sell intent
+		if ( 'sell' === intent ) {
```

Comment:
Last nitpick (I think) - isn't this duplicating the filtering that's already happening at https://github.com/Automattic/wp-calypso/blob/trunk/packages/design-picker/src/components/index.tsx#L255?

Reply:
... yes, yes you're right. 😅 Fixing that now!